Integrations/integration-RsaNetWitnessPacketsAndLogs.yml

```diff
@@ -63,32 +63,33 @@ script:
         return Object.keys(obj).length === 0;
     }
 
-    var mapTypeToValue = function (fields) {
-        var result = {};
-        var TYPE = 'type';
+    var fillgroupObject = function (fields, groupObject) {
+        var TYPE  = 'type';
         var VALUE = 'value';
+        var GROUP = 'group';
         if(fields && Array.isArray(fields)) {
             fields.forEach(function (element) {
-                if (element[TYPE] && element[VALUE]) {
-                    result[element[TYPE]] = element[VALUE];
+                if (element[TYPE] && element[VALUE] && element[GROUP]) {
+                    var index = parseInt(element[GROUP]);
+                    if(groupObject[index]) {
+                        groupObject[index][element[TYPE]] = element[VALUE];
+                    } else {
+                        groupObject[index] = {};
+                        groupObject[index][element[TYPE]] = element[VALUE];
+                    }
                 }
             });
         }
-        return result;
     }
 
     var buildHrAndContext = function (data) {
         //Check if we have values
         var retVal = {md:[], context: {NetWitness:{Events:[]}}};
-        var typeToValue;
+        var groupObject = [];
```

Comment:
you call it `groupObject` but it is an array. Call it something more simple `queryResults` or something similar.